### PR TITLE
Tighten up the docs

### PR DESCRIPTION
**Pull request by dafeder**
Some improvements to the docs structure to eliminate redundancy and improve readability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,38 @@
...
+It is also possible to run most DKAN Tools commands using a local webserver, database and PHP setup, but this practice is less supported.
+
 ## Installation
 
 1. Download or clone this repository into any location on your development machine.
-2. Add _/bin/dktl_ to your path somehow. This is often accomplished by adding a symbolic link to a folder already in your path, like _~/bin_. For instance: 
+2. Add _bin/dktl_ to your `$PATH` somehow. This is often accomplished by adding a symbolic link to a folder already in your path, like _~/bin_. For instance: 
 
 ```bash
 ln -s /my/dktl/location/bin/dktl ~/bin/dktl
```
Comment (janette) [inline README.md]: that 'location' bit might be confusing to the literal-minded, how about `ln -s /myworkspacelocation/dktl/bin/dktl ~/bin/dktl

File: README.md [hunk 2]
```diff
@@ -53,30 +65,34 @@ The `make` command will get all of DKAN's dependencies _including_ Drupal core.
...
+To accomplish this, DKAN Tools projects will have the following basic directory structure, created when you run `dktl init`.
 
-```bash
-mkdir my_project && cd my_project
-dktl dc up -d && dktl init
-dktl dkan:get <version_number>
-```
+    ├── dkan              # The upstream DKAN core codebase
+    ├── docroot           # Drupal core, and contrib modules not from DKAN
+    ├── src               # Site-specific configuration, code and files
```
Comment (janette) [inline README.md]: Let's also add the 'patches' directory here

File: README.md [hunk 3]
```diff
@@ -87,27 +103,21 @@ projects:
...
 ```
 
-In _/src/make/drupal.make_ we can also define the contributed modules, themes, and libraries that our site uses. For example if our site uses the deploy module we can add this to _/src/make/drupal.make_ under the `projects` section:
+In _/src/make/drupal.make_ we can also define the contributed modules, themes, and libraries that our site uses. For example if our site uses the [Deploy](https://www.drupal.org/project/deploy) module we can add this to _/src/make/drupal.make_ under the `projects` section:
 
 ```yaml
 projects:
   deploy:
     version: '3.1'
 ```
```
Comment (janette) [inline README.md]: If our site required a custom patch to the deploy module, we add a _src/patches/custom_patch.patch_ directory. For public patches we just need the url to the patch.
```yaml
projects:
    deploy:
        version: '3.1'
        patch:
            1: '../patches/custom_patch.patch
            3005415: 'https://www.drupal.org/files/issues/2018-10-09/use_plain_text_format-3005415.patch'
```

File: README.md (hunk 2 above)
Comment (dafeder) [reply README.md]: ok I'll add a mention of both _patches_ and _backup_

**Comment by dafeder** [comment]
Made edits requested

File: README.md [hunk 4]
```diff
@@ -1,28 +1,38 @@
...
 
 1. Download or clone this repository into any location on your development machine.
-2. Add _/bin/dktl_ to your path somehow. This is often accomplished by adding a symbolic link to a folder already in your path, like _~/bin_. For instance: 
+2. Add _bin/dktl_ to your `$PATH` somehow. This is often accomplished by adding a symbolic link to a folder already in your path, like _~/bin_. For instance, if DKAN Tools is located in _/myworkspace_: 
+
+```bash
+ln -s  /myworkspace/dkan-tools/bin/dktl ~/bin/dktl
+```
+
+Alternatively, you could add _/my/dktl/location/bin_ directly to your `$PATH`. Enter this in your terminal or add it to your session permanently by adding a line in _.bashrc_ or _.bash_profile_:
```
Comment (janette) [inline README.md]: @dafeder should this be _/myworkspace/dkan-tools/bin_ to be consistent?

**Pull request merged**